test/__snapshots__/storageApi.test.ts.snap

```diff
@@ -50,6 +50,16 @@ Object {
 }
 `;
 
+exports[`create new public bucket 1`] = `
+Object {
+  "created_at": "2021-05-31T23:05:11.103209+00:00",
+  "id": "my-new-public-bucket-1622502310904",
+  "name": "my-new-public-bucket-1622502310904",
+  "owner": "317eadce-631a-4429-a0bb-f19a7a517b4a",
+  "updated_at": "2021-05-31T23:05:11.103209+00:00",
```

Comment:
Hey @inian ! 

I was hoping to be able to verify that the bucket created with public option is indeed public, but I don't see a public field in a returned value. Are there any ways of verifying if a bucket is public or not?

Reply:
Hey Tyler, yup we don't return the bucket visibility on bucket create response but you should get it when you do a GET /bucket/bucketid.